src/utils/types.ts

```diff
@@ -107,9 +108,10 @@ interface EvaluatorParameters {
   APIfetch?: Function
 }
 
-interface FullUserPermissions {
+interface FullUserInfo {
   username: string
```

Comment:
technically speaking this wont' be needed anymore

Reply:
💨 Gone